R/navs-aliases.R

```diff
@@ -0,0 +1,51 @@
+#' @include nav-items.R navs-legacy.R navs.R
+
+# nav-items Aliases for future deprecation ----------------------------------
+
+# TODO: Deprecate `nav()`` in a future version of {bslib}
+#' @describeIn nav An alias for `nav_panel()`.
+#' @export
+nav <- nav_panel
```

Comment:
Currently, these "soon-to-deprecated" names will still show up in the function reference. Seems better if we put them all on the same `#' @keywords internal` page and mention that they'll be deprecated/superseded soon.

Now that I'm saying that out-loud though, that's a bit weird...I wouldn't be opposed to actually deprecating the old names now instead of later (see `R/deprecated.R`)

Reply:
Done! I added them to `R/deprecated.R` and included a small section in the docs there with an updated naming guide. I assumed our next release will be `v0.5.0` and included that in the docs; if that's not the case we'd need to update that text.